lib/static/components/state/state-error.jsx

```diff
@@ -56,8 +84,17 @@ class StateError extends Component {
             let content = '';
 
             if (typeof value === 'string') {
+                if (key === "snippet") {
+                    value = this._tryToImproveSnippet(value);
+                }
+
                 if (value.match(/\n/)) {
                     [titleText, ...content] = value.split('\n');
+
+                    if (!content[0].length) {
+                        content = content.slice(1);
+                    }
```

Comment:
`Expect`-s have extra empty line in error message, which could be removed:

### Before
<img width="379" alt="image" src="https://github.com/gemini-testing/html-reporter/assets/54893992/dd2cbe13-754d-4c12-842e-030abf44d9df">


### After
<img width="379" alt="image" src="https://github.com/gemini-testing/html-reporter/assets/54893992/3744dce0-439e-495b-a363-db77e23c6685">